SPC Categorial Outlook Display.py

```diff
@@ -38,80 +42,64 @@ def display_spc_outlook(outlook_data):
             ax.add_patch(mpatches.Polygon(list(zip(x, y)), alpha=0.5, ec='k', lw=1, fc=get_outlook_color(outlook_type)))
 
     # Overlay US state outlines
-    
-    ### ENTER STATE OUTLINE FILE DIRECTORY BELOW (Use / and not \) ###
-    states = gpd.read_file('') 
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    states_shapefile = os.path.join(current_directory, 's_11au16.shp')  
+    states = gpd.read_file(states_shapefile)  

```

Comment:
Edited file management to better read the file from different machines based off the same install